test/integration/consul-container/libs/utils/version_oss.go

```diff
@@ -12,3 +12,4 @@ var TargetImage = flag.String("target-image", "consul", "docker image name to be
 var TargetVersion = flag.String("target-version", "local", "docker image version to be used as UUT (unit under test)")
 var LatestImage = flag.String("latest-image", "consul", "docker image name to be used under test (Default: consul)")
 var LatestVersion = flag.String("latest-version", "1.11", "docker image to be used as latest")
+var FollowLog = flag.Bool("follow-log", false, "follow container log in output")
```

Comment:
Thanks. I will create a follow-up task to import the upgrade test to ENT

Reply:
@DanStough , just want to make sure I understand you correct. Are you saying the ` compatibility-integration-test:` in ENT still uses the OSS image (the link is omitted for privacy)? Please correct me if I am wrong. Thanks.

Reply:
I just wanted to make sure this line gets added to `test/integration/consul-container/libs/utils/version_ent.go` in case we want to run enterprise-specific tests. It just so happens it isn't different from this file and I don't think we use it. TBH we could just get rid of the build tags and rename this file `version.go`.

Reply:
Got it, @DanStough . 

@eculver  is working on enabling the upgrade test in ENT and we will pay attention to this. Thanks.